site/search/useSearchChartHitData.ts

```diff
@@ -1,166 +0,0 @@
-import { useMemo } from "react"
```

Comment:
Some of the functions have been moved to the RichData component file, others are not needed anymore.